client/src/containers/App/PublicContainer.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+import React, { Fragment, Component } from 'react';
+import Events from '../Events/Events';
+import { EMNavbar } from '../../components/EMNavbar';
+import type { BaseReduxPropTypes } from '../../types/base-props-types';
+import { Container } from 'reactstrap';
+import { Route, BrowserRouter as Router } from 'react-router-dom';
+import EventDetails from '../EventDetails/EventDetails';
+import Organisation from '../Organisation/Organisation';
+import { connect } from 'react-redux';
+
+type Props = {};
+
+class PublicContainer extends Component<Props> {
+  render() {
+    return (
+      <Router>
+        <Fragment>
+          <Route
+            exact
+            render={() => (
```

Comment:
@QaseemLodhi we dont want to pass in EMNavbar with every route. need to refactor this further